src/Hazelcast.Net/CP/ICP.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Threading.Tasks;
+
+namespace Hazelcast.CP
+{
+    /// <summary>
+    /// Defines the CP subsystem.
+    /// </summary>
+    public interface ICP
```

Comment:
All other clients name this as `ICPSubsystem`, can we use the same naming here as well?